yosys.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+cd /src
```

Comment:
This script only will work if run in jenkins.  Its probably better to move it somewhere else.  Or move it into the docker image repo?

Reply:
@stffrdhrn It will generate printing statistics and can work in Travis also, at least to show the results in travis.log. Do you think I should add this to Travis too?